_includes/manuals/nightly/4.1.3_trends.md

```diff
@@ -1,12 +1,3 @@
 
-Trends in Foreman allow you to track changes in your infrastructure over time. It allows you to track both Foreman related information and any puppet facts. The Trend pages give a graph of how the number of hosts with that value have changed over time, and the current hosts list.
-
-There are two pieces to the Trends area, the Trends to track and their corresponding counters. To define trend counters, use the "Add Trend Counter" button on the '/trends' page. Optionally set the "Name" field to over-ride odd puppet fact names to be more readable. Once created you can optionally 'Edit' the Trend to change the display names of the underlying values.
-
-Next, to start collecting trend data, set a cron job to execute 'foreman-rake trends:counter'. Each time the rake task executes it will create 1 tick on the graphs, so you can fine tune the granularity with your cron job. We recommend using the same as your puppet run interval (30 minutes). Here's an example to run once per hour:
-
-{% highlight sh %}
-0 * * * * /usr/sbin/foreman-rake trends:counter
-{% endhighlight %}
-
-Finally note that these trends are the same for all users. So if you delete a Trend category you will loose all history for that trend and the trackers will start all over again. So please be careful when deleting.
+Trends and statistics are moved to the separate plugin.
+See [manual](/plugins/foreman_statistics) for the reference.
```

Comment:
can you also add this in the deprecations for 2.1 manual and upgrade warning section for nightly manual?

Reply:
Done :)